src/Reflection/ReflectionObject.php

```diff
@@ -0,0 +1,465 @@
+<?php
+
+namespace BetterReflection\Reflection;
+
+use BetterReflection\Reflector\ClassReflector;
+use BetterReflection\SourceLocator\EvaledCodeSourceLocator;
+use PhpParser\Node\Stmt\Property as PropertyNode;
+use PhpParser\Builder\Property as PropertyNodeBuilder;
+
+class ReflectionObject extends ReflectionClass
```

Comment:
In order to avoid future breakages (since you designed this class as a decorator, but then used inheritance due to how Reflection works in internals) you need a test that verifies that ALL public methods of this class are declared in this class.